roles/kubernetes-apps/istio/defaults/main.yml

```diff
@@ -0,0 +1,32 @@
+---
+istio_enabled: false
```

Comment:
can you copy this to example/group_vars/k8s-cluster.yaml ?

Reply:
Sure